examples/updateTranslationJob.py

```diff
@@ -44,8 +44,8 @@
     debug=True
 )
 
-# Update a job that has an id of 42, and reject it, cite the reason,
-# add a comment, and throw up some captcha stuff. See the docs for
+# Update a job that has an id of 42, and reject it, cite the reason
+# and add a comment. See the docs for
```

Comment:
you forgot that line https://github.com/gengo/gengo-python/blob/7364afdff49accf82988f4176fdc4730d614e942/examples/updateTranslationJob.py#L54

Reply:
Fixed in https://github.com/gengo/gengo-python/pull/103/commits/05393b2063119433887ab4966275816cb8c3baba